client/src/App.js

```diff
@@ -21,9 +22,10 @@ export default class App extends Component {
   render() {
     return (
       <div>
-        <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
-      </div>
+      <SavedList list={this.state.savedList} />
+      <Route path='/' exact component={MovieList} />
+      <Route path='/movies/:id' component={Movie} />
```

Comment:
We want to pass the `addToSavedList` method to `<Movie />`. A way to pass props to a `<Route />` is to use the render props technique 
```suggestion
      <Route
          path="/movies/:id"
          render={props => (
            <Movie {...props} addToSavedList={this.addToSavedList} />
          )}
        />
```